5-documentation/guides/agents/intermediate/local-agent-inspector/agent_inspector_example.py

```diff
@@ -0,0 +1,30 @@
+from uagents import Agent, Context, Model
+from uagents.setup import fund_agent_if_low
+
+
+class Message(Model):
+    message: str
+
+
+bob = Agent(
+    name="Bob",
+    port=8001,
+    seed="BobSecretPhrase",
+    endpoint=["http://127.0.0.1/:8001/submit"],
+)
+
+fund_agent_if_low(bob.wallet.address())
```

Comment:
This should not be necessary anymore.
We as long as an agent is on the test-net it will automatically be funded - we just need to make it clear that agents that are running on main-net will need to have tokens supplied to their wallet to work.